src/ubuntu/18.04/msquic/amd64/Dockerfile

```diff
@@ -18,11 +18,12 @@ RUN curl -OL https://packages.microsoft.com/config/ubuntu/18.04/packages-microso
         git \
         liblttng-ust-dev \
         lttng-tools \
+        libssl-dev \
```

Comment:
Please place this in alphabetical order with the other package names.

Reply:
Done